spec/models/destination_spec.rb

```diff
@@ -1,5 +1,24 @@
 require 'rails_helper'
 
 RSpec.describe Destination, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  subject do
+    Destination.create(name: 'London', location: 'London', price_per_day: 25,
+                       image_url: 'https://via.placeholder.com/150')
+  end
+
+  it 'create a valid destination object' do
+    expect(subject).to be_valid
+  end
+
```

Comment:
Same thing as with Reservation tests. You can remove this piece of code, regarding that you are using shoulda gem.

Reply:
Thanks @ShinobiWarior . I've fixed it.